docs/CHANGELOG.md

```diff
@@ -33,6 +35,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * Updated `specs` to `0.14` and `specs-hierarchy` to `0.3`. ([#1122])
 * Updated `winit` to `0.18` (see [Winit's changelog][winit_018]). ([#1131])
 * Updated `glutin` to `0.19` (see [Glutin's changelog][glutin_019]). ([#1131])
+* Renamed the `DrawSprite` pass to `DrawFlat2D` as it now handles both sprites and images without spritesheets. ([#1153])
 
```

Comment:
Add a link pointing to this pull request

`[#1153]: https://github.com/amethyst/amethyst/pull/1153`